File: competitive-events-api/src/main/java/es/jocasolo/competitiveeventsapi/dto/eventuser/EventUserPostDTO.java
package es.jocasolo.competitiveeventsapi.dto.eventuser;

import java.io.Serializable;

import es.jocasolo.competitiveeventsapi.dto.DTO;
import lombok.Getter;
import lombok.Setter;

public class EventUserPostDTO extends DTO implements Serializable {

	private static final long serialVersionUID = 1L;

	@Getter
	@Setter
	private String username;

	@Getter
	@Setter
	private Boolean reject = false; // Reject invitation to join event

	@Override
	public String toString() {
		return String.format("EventUserPostDTO [user=%s]", username);
	}

}

File: competitive-events-app/app/src/main/java/es/jocasolo/competitiveeventsapp/dto/eventuser/EventUserPostDTO.kt
package es.jocasolo.competitiveeventsapp.dto.eventuser

import com.google.gson.annotations.SerializedName

class EventUserPostDTO(
        @SerializedName("username") var username: String,
        @SerializedName("reject") var reject: Boolean? = null,
        @SerializedName("phone") var phone: String? = null,
        @SerializedName("email") var email: String? = null
) {

}